prog/vm/nexus.rb

```diff
@@ -176,6 +188,20 @@ def create_ipsec_tunnel(my_subnet, dst_vm, dst_subnet)
 
   def run
     host.sshable.cmd("sudo systemctl start #{q_vm}")
+    hop :setup_sshable
+  end
+
+  def setup_sshable
+    Sshable.create(host: vm.ephemeral_net6.network.to_s + "2") { _1.id = vm.id }
```

Comment:
not sure if this is a must, I may endup implementing a utility function to ssh into the vm instead of creating an sshable.